tests/serve_app/oxo_test.py

```diff
@@ -3092,3 +3094,33 @@ def testPublishAgentGroup_withoutNameAndAgentArgs_shouldPersistAgentGroup(
     assert len(ag["agents"]["agents"]) == 1
     assert ag["agents"]["agents"][0]["key"] == "agent_key"
     assert len(ag["agents"]["agents"][0]["args"]["args"]) == 0
+
+
+def testQueryScan_always_shouldReturnScanWithAgentGroup(
+    authenticated_flask_client: testing.FlaskClient, scan_with_agent_group: models.Scan
+) -> None:
+    """Test query for scan with agent group."""
+
+    query = """
+        query scan($scanId: Int) {
+  scan(scanId:$scanId) {
+    id
+    agentGroup{
```

Comment:
agent, args, values where are those to ensure all is working?

Reply:
applied